packages/api/emails/surveySocialWorker/subject.pug

```diff
@@ -0,0 +1 @@
+= `STEP 3 EMAIL TO SOCIAL WORKER`
```

Comment:
Based on the template I think the subject should be ` Step 3 Supporting Survey Sent to Family Representative`

Reply:
Done!